src/pocketmine/entity/Entity.php

```diff
@@ -982,6 +983,17 @@ public function getDataPropertyManager() : DataPropertyManager{
 		return $this->propertyManager;
 	}
 
+	/**
+	 * Called when interacted or tapped by a Player
+	 *
+	 * @param Player  $player
+	 * @param Item    $item
+	 * @param Vector3 $clickPos
+	 * @param int     $slot
+	 */
+	public function onInteract(Player $player, Item $item, Vector3 $clickPos, int $slot) : void{
+	}
```

Comment:
Closing bracket is on the next line following PSR-2 Standards with exceptions to The PocketMine-MP Code Syntax Standards in https://github.com/pmmp/PocketMine-MP/blob/master/CONTRIBUTING.md > "Opening braces MUST go on the same line, and MUST NOT have spaces before."

Reply:
there should be an empty line after `public function ...`, see other methods for examples.